OpenRA.Game/Support/AssemblyLoader.cs

```diff
@@ -9,9 +9,6 @@
  */
 #endregion
 
-// Not used/usable on Mono. Only used for Dotnet Core.
-// Based on https://github.com/natemcmaster/DotNetCorePlugins and used under the terms of the Apache 2.0 license
-#if NET5_0_OR_GREATER
```

Comment:
There some other `#if NET5_0_OR_GREATER` and `#if !NET5_0_OR_GREATER` statements across the codebase that can be simplified as well.